src/patterns/step-by-step-navigation/index.md.njk

```diff
@@ -0,0 +1,124 @@
+---
+title: Step by step navigation
+description: A starting point for your digital service on GOV.UK
+section: Patterns
+theme: GOV.UK Publishing
+aliases:
+backlog_issue_id: 111
+layout: layout-pane.njk
+---
+
+Step by step navigation is a way of presenting end-to-end journeys on GOV.UK. It groups existing guidance and transactions into a series of steps.
```

Comment:
There's lots of mention of 'GOV.UK', my definition of GOV.UK is government online, which includes services teams.

Should we add some nuance to this to ensure this is www.GOV.UK or some other way of saying 'the part of GOV.UK maintained by GDS that publishes the entry points to services'?

If this is too hand wavey feel free to disregard